code/options/OptionsManager.cpp

```diff
@@ -40,15 +40,14 @@ std::unique_ptr<json_t> OptionsManager::getValueFromConfig(const SCP_string& key
 {
 	auto override_iter = _config_overrides.find(key);
 	if (override_iter != _config_overrides.end()) {
-		// We return a reference to an existing object so we need to increment the reference count
-		json_incref(override_iter->second.get());
-		return std::unique_ptr<json_t>(override_iter->second.get());
+		json_t temp = *override_iter->second.get();
```

Comment:
This creates a copy of data that we do not have a full definition of at this point (for example, see the `json_object_t` type to see what the actual structure of this looks like when we are dealing with an object.

In short, this is not how you copy an instance of `json_t`.